src/interfaces/index.ts

```diff
@@ -7,3 +7,6 @@ export * from './store-interfaces/state-interface'
 export * from './store-interfaces/action-interface'
 export * from './store-interfaces/tombstone-interface'
 export * from './store-interfaces/getter-interface'
+
+// Others
+export * from './filingData-interface'
```

Comment:
I like it! As far as directory structure ( thinking ahead ), would this fall into any interface type category that we might add too?

Reply:
I'm not sure yet, which is why I didn't create one :)

I expect this will either be clearer in the future or not matter :)

Reply:
Sounds good, true agile ;)